modules/tinymce/src/core/main/ts/api/util/Observable.ts

```diff
@@ -47,7 +47,7 @@ const Observable: Observable<any> = {
    * @param {Boolean?} bubble True/false if the event is to be bubbled.
    * @return {Object} Event args instance passed in.
    * @example
-   * instance.fire('event', {...});
+   * editor.fire('event', {...});
```

Comment:
We can't guarantee this will be the editor. As an example, this is also used as a mixin for the `EditorManager`.